app/views/change_existing_user_requests/_request_details.html.erb

```diff
@@ -1,5 +1,19 @@
 <p><%= f.object.class.description %></p>
 
+<div class="alert alert-info alert-block">
+  <p>
+    You do not need to submit a request if you're a super organisation admin or organisation admin and you're requesting to:
+    <ul>
+    <li>change an account's permissions for a 'devolved' application, including upgrading an account to 'editor' in Whitehall Publisher</li>
+    <li>reset 2-step verification for an account</li>
+    <li>unsuspend an account</li>
+    <li>resend a signup ('account invitation') email for an account</li>
+    </ul>
+
```

Comment:
```suggestion
</p>
<p>
```

Reply:
Looks like there's no style defined for the <p> tag that set the margins so adding this makes no visual difference. Peter added a line break instead.